apps/workers/openaiWorker.ts

```diff
@@ -97,12 +97,17 @@ with the key "tags" and the value is list of tags. Don't wrap the response in a
 
 const TEXT_PROMPT_BASE = `
```

Comment:
I wish was possible to customise TEXT_PROMPT_BASE directly within the settings menu in all apps.

Great idea a nice job so far!

Reply:
I don't want to give the people the freedom to add whatever instructions they want to be honest because it limits my ability to change or improve the prompt in the future. What I can do instead, is maybe allow the users to add some extra instructions at the end.